recipes/geotrans/all/conandata.yml

```
@@ -1,7 +1,12 @@
 sources:
+  "3.9":
+    url: "https://earth-info.nga.mil/php/download.php?file=wgs-mastertgz"
```

```suggestion
    url: "https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/geotrans/geotrans-3.9.tgz"
```

I just uploaded a new mirror into Artifactory. The SHA-256 is the very same, of course.